app/src/main/java/com/example/calculator/Calculate.kt

```diff
@@ -0,0 +1,97 @@
+package com.example.calculator
+
+import java.util.ArrayDeque
+
+class Calculate {
+    private var list : MutableList<Any> = mutableListOf()
+
+    fun digitsOperator(txtInput : String) {
+        var currentDigit = ""
+        for (character in txtInput) {
+            if (character.isDigit() || character == '.') {
+                currentDigit += character
+            } else  {
+                list.add(currentDigit.toFloat())
+                list.add(character)
+                currentDigit = ""
+            }
+        }
+        if (currentDigit != "") {
+            list.add(currentDigit.toFloat())
+        }
+    }
+
+    fun clearList() {
+        list.clear()
+    }
+
+    fun result() : Float {
+        return evaluatePostfix()
+    }
+
+
+    // kiem tra do lon cua dau cong tru nhan chia
+    private fun priority(operator : Any) : Int {
```

Comment:
bắt đầu bằng động từ
```suggestion
    private fun getPriority(operator : Any) : Int {
```